src/main/java/org/jnode/fs/xfs/XfsDirectory.java

```diff
@@ -33,13 +35,10 @@ public class XfsDirectory extends AbstractFSDirectory implements FSDirectoryId {
     /**
      * The related entry.
      */
+    @NonNull
```

Comment:
Any reason to not use `javax.annotation.Nonnull`?

Reply:
yeah, the reason I didn't notice it :face_palm, will change it soon.